backend/main/server/controllers.go

```diff
@@ -353,6 +367,17 @@ func (a *App) updateCommunity(w http.ResponseWriter, r *http.Request) {
 	respondWithJSON(w, http.StatusOK, c)
 }
 
+func validateConractThreshold(s []models.Strategy) error {
```

Comment:
Could we put this at the bottom of the file with the helpers?

Reply:
sure thing!